src/coreclr/vm/jitinterface.cpp

```diff
@@ -6895,6 +6895,23 @@ static bool getILIntrinsicImplementationForUnsafe(MethodDesc * ftn,
 
         return true;
     }
+    else if (tk == CoreLibBinder::GetMethod(METHOD__UNSAFE__BYREF_IS_ADDRESS_GREATER_THAN_OR_EQUAL_TO)->GetMemberDef())
```

Comment:
This does not hurt, but it is not really needed given that the JIT expands it as intrinsic and there is fallback implementation.

Reply:
I simply based it off https://github.com/dotnet/runtime/pull/89156 - I can remove it if you'd like

Reply:
Also, I added fallback implementation to support mono - if we intrinsicified on mono, presumably, it would not be necessary any more.

Reply:
> I can remove it if you'd like 

Less code is generally better.

> if we intrinsicified on mono, presumably, it would not be necessary any more.

We would still need something for CoreCLR interpreter that does very limited intrinsic expansion